fedimint-server/src/config.rs

```diff
@@ -80,10 +82,10 @@ pub struct ServerConfigConsensus {
 pub struct ServerConfigLocal {
     /// Our peer id (generally should not change)
     pub identity: PeerId,
-    /// Our bind address for running HBBFT
-    pub hbbft_bind_addr: SocketAddr,
+    /// Our bind address for communicating with peers
+    pub fed_bind: SocketAddr,
```

Comment:
p2p_bind ?

Reply:
I think it does. We have bunch of other things with "p2p" in it. This is the port for p2p connections between federation members. All checks out to me.